pkgs/development/python-modules/libsixel/default.nix

```
@@ -0,0 +1,30 @@
+{ buildPythonPackage
+, lib
+, libsixel
+}:
+
+buildPythonPackage rec {
+  version = libsixel.version;
+  pname = "libsixel";
+
+  src = libsixel.src;
+  sourceRoot = "source/python";
```

To avoid issues if other package uses different src urls
```suggestion
  sourceRoot = "${src.name}/python";
```

---

Fixed